chia/wallet/wallet_state_manager.py

```diff
@@ -1196,6 +1202,25 @@ async def handle_clawback(
             await self.tx_store.add_transaction_record(tx_record)
         return None
 
+    async def handle_vc(self, parent_coin_spend: CoinSpend) -> Optional[WalletIdentifier]:
```

Comment:
@ytx1991 why is handling VCs in this PR? this should be removed

Reply:
It is in the main